src/main/java/org/javawebstack/orm/query/Query.java

```diff
@@ -164,6 +164,14 @@ public Query(Repo<T> repo, Class<T> model) {
         where.notNull(left);
         return this;
     }
+    
+    public Query<T> whereNull(Object left) {
+        return where.isNull(left);
```

Comment:
I'm not very familiar with this part of the code but it seems to me it should be:
```suggestion
        where.isNull(left);
        return this;
```

Reply:
This should work fine

Reply:
His code is totally fine and smaller